server/mounts.go

```diff
@@ -66,8 +66,8 @@ func (s *Server) customMounts() []environment.Mount {
 		}
 
 		// Check if the target path includes /home/container
-		if strings.Contains(target, "/home/container") {
-			logger.WithField("invalid_target_path", target).Warn("skipping custom server mount, target path includes /home/container")
+		if strings.Contains(target, "/home/container") && config.Get().BlockBaseDirMount {
```

Comment:
Btw. should this check if the string starts with `/home/container` instead of just contains?

Someone might have a path like `/some/path/home/container/something` and I think it's not something you want to block (without changing the config).

Reply:
as the target? normaly people keep it easy and use something like `/mount` or `/files` not a full path as the target.

I have never seen anyone use it like that so I do not see it as a real issue to be honnest

Reply:
I agree that it's some edge-case that might never occur, but just wanted to point this out, because I noticed it.

If go language has some equivalent of function "startsWith" then it should be a very quick fix that might be worth it. However if it's not easy to make such check then probably it's not worth it, especially because the config exists.

Just wanted to point out such edge-case. I'm not saying it's worth fixing, because I'm not familiar with go language, so it's hard to tell for me whether it's something that can be changed in 30 seconds or 30 minutes.

Reply:
the strings libary has a build in function for that so at max 30 seconds.

Reply:
`strings.HasPrefix` would be the way to look at if a string starts with something.